ndb_api/profile_helpers.go

```diff
@@ -117,3 +129,7 @@ var NetworkOOBProfileResolver = func(p ProfileResponse) bool {
 var DbParamOOBProfileResolver = func(p ProfileResponse) bool {
 	return p.SystemProfile && p.Type == common.PROFILE_TYPE_DATABASE_PARAMETER
 }
+
+var DbParamInstanceOOBProfileResolver = func(p ProfileResponse) bool {
+	return p.SystemProfile && p.Type == common.PROFILE_TYPE_DATABASE_PARAMETER && p.Topology == common.TOPOLOGY_INSTANCE
+}
```

Comment:
Can we also add a comment here that explains your findings regarding the overloaded use of "topology" for db param profiles? Might save/help us in the future.